_includes/build/requesting_data_all_three.html

```diff
@@ -93,8 +96,8 @@ <h4>Request Headers</h4>
 
 <h4>cURL Command to check the job status</h4>
 	<pre><code>curl -X GET "https://api.bcda.cms.gov/api/v2/jobs/42" \
-	-H "accept: application/fhir+json" \
-	-H "Authorization: Bearer <span style="color: #046B99;">{access_token}</span>"</code></pre>
```

Comment:
We should keep the whitespace (tab) characters since they'll be rendered inside the `<pre>` tag
ex: 
Tab
<pre>curl -X GET "https://api.bcda.cms.gov/api/v2/jobs/42" \
	-H "accept: application/fhir+json" \\</pre>
No tab
<pre>curl -X GET "https://api.bcda.cms.gov/api/v2/jobs/42" \
-H "accept: application/fhir+json" \</pre>